Agents/ForecastingAgent/tests/test_integration.py

```diff
@@ -517,3 +517,174 @@ def test_create_tft_forecast(
         print(f'{k}: {round(v,5)}')
 
     print("All check passed.")
+
+
+#pytest.mark.skip(reason="")
+@pytest.mark.parametrize(
+    "derivation_input_set, dataIRI, input_chunk_length, with_unit, overwrite_forecast, ts_times, covariates, case",
+    [
+        (cf.DERIVATION_INPUTS_7, cf.IRI_TO_FORECAST_1, cf.DURATION_2, False, True, cf.TIMES, cf.COVARIATES_2, cf.TEST_CASE_13),
+        (cf.DERIVATION_INPUTS_7, cf.IRI_TO_FORECAST_1, cf.DURATION_2, False, False, cf.TIMES, cf.COVARIATES_2, cf.TEST_CASE_14),
+        (cf.DERIVATION_INPUTS_8, cf.IRI_TO_FORECAST_2, cf.DURATION_2, True, True, cf.TIMES, cf.COVARIATES_2, cf.TEST_CASE_15),
+        (cf.DERIVATION_INPUTS_8, cf.IRI_TO_FORECAST_2, cf.DURATION_2, True, False, cf.TIMES, cf.COVARIATES_2, cf.TEST_CASE_16),
+        (cf.DERIVATION_INPUTS_9, cf.IRI_TO_FORECAST_1, cf.DURATION_2, False, True, cf.TIMES, cf.COVARIATES_2, cf.TEST_CASE_17),
+        (cf.DERIVATION_INPUTS_9, cf.IRI_TO_FORECAST_1, cf.DURATION_2, False, False, cf.TIMES, cf.COVARIATES_2, cf.TEST_CASE_18),
+        (cf.DERIVATION_INPUTS_10, cf.IRI_TO_FORECAST_2, cf.DURATION_2, True, True, cf.TIMES, cf.COVARIATES_2, cf.TEST_CASE_19),
+        (cf.DERIVATION_INPUTS_10, cf.IRI_TO_FORECAST_2, cf.DURATION_2, True, False, cf.TIMES, cf.COVARIATES_2, cf.TEST_CASE_20)
+    ],
+)
+def test_create_prophet_covariates_forecast(
+    initialise_clients, create_example_agent, derivation_input_set, dataIRI, input_chunk_length,
+    with_unit, overwrite_forecast, ts_times, covariates, case
+):
+    """
```

Comment:
Copy and paste error

Reply:
Sorry, I don't think I see it yet. Could you clarify the problem?